language-server/modules/langserver-core/src/test/resources/completion/expression_context/config/function_call_expression_ctx_config22.json

```diff
@@ -505,7 +506,7 @@
       "label": "Thread",
       "kind": "TypeParameter",
       "detail": "Union",
-      "sortText": "AP",
+      "sortText": "CP",
```

Comment:
Seems like this test case have been affected. Expected type for type param cases are incorrect.

Reply:
https://github.com/ballerina-platform/ballerina-lang/issues/39407